examples/maze-solver/maze_tests.py

```diff
@@ -43,10 +43,10 @@ def fill_with_zeros(solution_dict, n_rows, n_cols, ignore_list=None):
 
 def get_energy(solution_dict, bqm):
     min_energy = float('inf')
-    aux_variables = [v for v in bqm.variables if match("aux\d+$", v)]
+    aux_variables = [v for v in bqm.variables if re.match("aux\d+$", v)]
```

Comment:
What if `v` happens to be `int`, not `str`?

Reply:
Since `get_energy(..)` is meant only for the maze and since maze's variables all follow a specific string format (i.e. `r'^(\d+),(\d+)[nw]$'`), I think this `aux_variables ...` implementation is okay. However, if you disagree, please let me know.